app/views/payment/v2/StudentLicensesBodyComponent.vue

```diff
@@ -0,0 +1,374 @@
+            class="school-district__body row school-district__body-only"
+          >
+            <div class="col-md-6">
+              <includes-body-component class-type="school-district" />
+            </div>
+            <div class="col-md-6">
+              <addon-body-component class-type="school-district" />
+            </div>
+          </div>
+          <body-footer-component :class-type="'school-district'" :num-students="numStudents" />
+        </div>
+      </div>
+    </div>
+    <div class="footer">
+      <div class="footer__text">
+        See also our <a href="https://www.ozaria.com/funding" target="_blank">Funding Resources Guide</a> for how to leverage CARES Act funding sources like ESSER and GEER.
+      </div>
+    </div>
+    <modal-get-licenses
+      v-if="openLicenseModal"
+      @close="openLicenseModal = false"
+    />
+    <purchase-license-modal
+      :payment-group="paymentGroup"
+      v-if="openPurchaseModal"
+      @close="openPurchaseModal = false"
+    />
+    <backbone-modal-harness
+      :open="openApplyLicenseModal"
+      :modal-view="HowToEnrollModal"
+      @close="openApplyLicenseModal = false"
+    />
+  </div>
+</template>
+
+<script>
+import IncludesBodyComponent from './student-license/IncludesBodyComponent'
+import AddonBodyComponent from './student-license/AddonBodyComponent'
+import BodyFooterComponent from './student-license/BodyFooterComponent'
+import ModalGetLicenses from '../../../components/common/ModalGetLicenses'
+import PurchaseLicenseModal from './student-license/PurchaseLicenseModal'
+import BackboneModalHarness from '../../common/BackboneModalHarness'
+import HowToEnrollModal from '../../teachers/HowToEnrollModal'
+export default {
+  name: 'StudentLicensesBodyComponent',
+  components: {
+    IncludesBodyComponent,
+    AddonBodyComponent,
+    BodyFooterComponent,
+    ModalGetLicenses,
+    PurchaseLicenseModal,
+    BackboneModalHarness
+  },
+  data () {
+    return {
+      openLicenseModal: false,
+      openPurchaseModal: false,
+      openApplyLicenseModal: false,
+      HowToEnrollModal
+    }
+  },
+  props: {
+    numStudents: {
+      type: String,
+      required: true,
+      validator: (v) => {
+        return [ '<=10', '10+' ].includes(v)
+      }
+    },
+    paymentGroup: {
+      type: Object
+    }
+  },
+  methods: {
+    onSchoolDistrictClicked () {
+      this.openLicenseModal = true
+    },
+    onSmallClassClicked () {
+      this.openPurchaseModal = true
+    },
+    onApplyLicenseClicked () {
+      this.openApplyLicenseModal = true
+    }
+  }
+}
+</script>
+
+<style scoped lang="scss">
+.student-licenses-body {
+  font-size: 62.5%;
+}
+.license {
+  background: #FFFFFF;
+  /* Gray 3 */
+
+  border: 1px solid #D8D8D8;
+  box-sizing: border-box;
+  /* Drop Shadow (Box) */
+
+  box-shadow: 3px 0 8px rgba(0, 0, 0, 0.15), -1px 0 1px rgba(0, 0, 0, 0.06);
+  border-radius: 2.4rem;
+
+  overflow: hidden;
+
+  font-family: 'Work Sans', serif;
```

Comment:
I know we haven't done this in many other places throughout the code, but I'm not sure how reliably we will have loaded Work Sans; might it make sense to use a font stack with Open Sans as well? I would usually put the font stack in a variable, like this: https://github.com/codecombat/codecombat/blob/master/app/styles/home-view.sass#L7

Also, if Work Sans is not available, use sans serif, not serif.